src/ArrowTypes/Project.toml

```diff
@@ -18,8 +18,7 @@
 name = "ArrowTypes"
 uuid = "31f734f8-188a-4ce0-8406-c8a06bd891cd"
 authors = ["quinnj <quinn.jacobd@gmail.com>"]
-version = "2.0.1"
-
+version = "2.0.2"
```

Comment:
I considered making this a minor release due to `ArrowTypes.default` not being fully documented

Reply:
We haven’t registered 2.0.1 yet btw, so automerge will complain, but that’s ok and we can manually merge